multimedia/mat2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mat2
+version             0.13.3
+categories          multimedia
+license             LGPL-3
+supported_archs     noarch
+platforms           {darwin any}
+maintainers         {@akierig fastmail.de:akierig} \
+                    openmaintainer
+
+description         metadata removal tool
+long_description    mat2 is a metadata removal tool, supporting a wide range \
+                    of commonly used file formats, written in python3.
+homepage            https://0xacab.org/jvoisin/mat2
+master_sites        https://0xacab.org/jvoisin/mat2/-/archive/${version}/
+
+checksums           rmd160  69de0de9284552f001bd5f4aa2210b021a42eca9 \
+                    sha256  e1bb0161fa1c2f2adb1b933761f9569534309e90209568f96aca5fc8a74f511a \
+                    size    11782599
+
+python.versions     39 310
+
+depends_lib         port:py${python.version}-mutagen \
```

Comment:
this should become `depends_lib-append` as not to overwrite the dependency on the `pythonXY` port added by the `python` PG